### ISSUE TITLE
[FR] Support heading links

### ISSUE THREAD

**Issue by FeralFlora** [alangrainger/obsidian-share#10]
It seems that heading links are not supported? I tried to create a table of contents like so:

```markdown
- [[#Reasoning and workflow|Reasoning and workflow]]
	- [[#Reasoning and workflow#Here's the gist of this workflow:|Here's the gist of this workflow:]]
- [[#What Zotero Kanban Maker does|What Zotero Kanban Maker does]]
- [[#Setup|Setup]]
	- [[#Setup#Plugins required|Plugins required]]
	- [[#Setup#First steps|First steps]]
	- [[#Setup#QuickAdd setup|QuickAdd setup]]
	- [[#Setup#Recommended Kanban board settings|Recommended Kanban board settings]]
	- [[#Setup#One-click literature note imports and updates|One-click literature note imports and updates]]
	- [[#Setup#Zotero setup|Zotero setup]]
- [[#Usage summary|Usage summary]]
- [[#Get the gist?|Get the gist?]]
- [[#Future plans|Future plans]]
```

But the links were simply stripped, leaving only the link text: 
![image](https://github.com/alangrainger/obsidian-share/assets/33608083/b64f7f74-1eb7-4a3d-8c83-fb17a5774a5c)

**Comment by alangrainger** [comment]
Added in 0.3.9:

https://file.obsidianshare.com/87/8085a351595736266c54873c7295b96b.html#lkfo2v02dA2QUit2HlKwMJdQDzyAqR+W8gVJklp8UXA

**Comment by alangrainger** [comment]
Fixed in 0.3.18 - very quick bug:

https://github.com/alangrainger/obsidian-share/commit/04e0f065aa4d018b50370e0fba169f635f8c5dfa
